djlint/settings.py

```diff
@@ -1123,3 +1123,9 @@ def __init__(
               )
         """
         )
+
+    def get_resolved_exclude_paths(self) -> list[Path]:
+        exclude_paths = []
+        for p in self.exclude.split("|"):
+            exclude_paths.append(Path(p.strip().replace("\\.", ".")).resolve())
+        return exclude_paths
```

Comment:
Not sure but maybe this could also just be a comprehension :)

```suggestion
        return [Path(p.strip().replace("\\.", ".").resolve() for p in self.exclude.split("|")]
```

Reply:
Or even better a for loop with yield (generator)

Reply:
I made another attempt and pushed 1ccc3450cfddcdcbaa582674f6f23e7d635f690e